cylc/flow/util.py

```diff
@@ -15,20 +15,44 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """Misc functionality."""
 
+import ast
 from contextlib import suppress
 from functools import partial
 import json
 import re
+from textwrap import dedent
 from typing import (
     Any,
+    Callable,
     List,
     Sequence,
+    Tuple,
 )
 
+# U+2713 (check)
+# U+2A2F (vector cross product)
+BOOL_SYMBOLS: List[str] = ['⨯', '✓']
```

Comment:
Considering how this is used
```suggestion
BOOL_SYMBOLS: Dict[bool, str] = {
    False: '⨯',  # U+2A2F (vector cross product)
    True: '✓'  # U+2713 (check)
}
```

Reply:
In Python bools are integers, so what I;ve done is fine.

Reply:
I think this way makes it clearer what the intent is (also gets rid of the mypy complaint when trying to do `BOOL_SYMBOLS[False]`?)